src/workerd/api/node/BUILD.bazel

```diff
@@ -1,6 +1,20 @@
 load("//:build/kj_test.bzl", "kj_test")
 load("//:build/wd_cc_library.bzl", "wd_cc_library")
 
+wd_cc_library(
+    name = "node-exceptions",
+    srcs = [
+        "exceptions.c++",
```

Comment:
I think it's a bit confusing when the Bazel target name doesn't match the filenames. The file is just exceptions.c++ but the target is node-exceptions.